.vscode/tasks.json

```diff
@@ -32,7 +32,7 @@
         "command": "rm",
         "args": [
           "-rf",
-          "build"
+          "build/*"
```

Comment:
Why not delete the full build folder than? Should be a separate commit (nitpick)

Reply:
In case you want to link the `build` directory to a different location (e.g. ramdisk to speed up the build).